homeassistant/components/renson/services.yaml

```diff
@@ -0,0 +1,216 @@
+          max: 35
+          step: 1
+          unit_of_measurement: "°C"
+          mode: slider
+    activate:
+      name: Activate
+      description: Activate or disable the breeze feature
+      required: true
+      advanced: false
+      default: false
+      selector:
+        boolean:
+
+set_day_night_time:
+  name: Set day and night time
+  description: Set the day and night time of the system
+  fields:
+    day:
+      name: Start day
+      description: Start time of the day
+      required: true
+      advanced: false
+      default: false
+      selector:
+        time:
+    night:
+      name: Start night
+      description: Start time of the night
+      required: true
+      advanced: false
+      default: false
+      selector:
+        time:
+
+set_pollution_settings:
+  name: Set pollution settings
+  description: Set all the pollution settings of the ventilation system
+  fields:
+    day_pollution_level:
+      name: Day pollution Level
+      description: Ventilation level when pollution is detected in the day
+      required: false
+      advanced: false
+      default: "Level3"
+      selector:
+        select:
+          options:
+            - "Level1"
+            - "Level2"
+            - "Level3"
+            - "Level4"
+    night_pollution_level:
+      name: Night pollution Level
+      description: Ventilation level when pollution is detected in the night
+      required: false
+      advanced: false
+      default: "Level2"
+      selector:
+        select:
+          options:
+            - "Level1"
+            - "Level2"
+            - "Level3"
+            - "Level4"
+    humidity_control:
+      name: Enable humidity control
+      description: Activate or disable the humidity control
+      required: false
+      advanced: false
+      default: true
+      selector:
+        boolean:
+    airquality_control:
+      name: Enable air quality control
+      description: Activate or disable the air quality control
+      required: false
+      advanced: false
+      default: true
+      selector:
+        boolean:
+    co2_control:
+      name: Enable CO2 control
+      description: Activate or disable the CO2 control
+      required: false
+      advanced: false
+      default: true
+      selector:
+        boolean:
+    co2_threshold:
+      name: co2 threshold
+      description: Sets the CO2 pollution threshold level in ppm
+      required: false
+      advanced: false
+      default: 600
+      selector:
+        number:
+          min: 400
+          max: 2000
+          step: 50
+          unit_of_measurement: "ppm"
+          mode: slider
+    co2_hysteresis:
+      name: co2 hysteresis
+      description: Sets the CO2 pollution threshold hysteresis level in ppm
+      required: false
+      advanced: false
+      default: 100
+      selector:
+        number:
+          min: 50
+          max: 400
+          step: 50
+          unit_of_measurement: "ppm"
+          mode: slider
+
+set_filter_days:
```

Comment:
Can be a `Number` entity?

Reply:
I created a number entity for this https://github.com/home-assistant/core/pull/99358